app/Commands/KnowledgeServeCommand.php

```
@@ -0,0 +1,444 @@
+        // Resolve docker service at runtime for better testability
+        $this->docker = app(DockerServiceInterface::class);
+
+        $action = $this->argument('action');
+
+        return match ($action) {
+            'install' => $this->install(),
+            'start' => $this->start(),
+            'stop' => $this->stop(),
+            'status' => $this->status(),
+            'restart' => $this->restart(),
+            default => $this->invalidAction($action),
```

PHPStan (2.1.33)

Parameter #1 $action of method App\Commands\KnowledgeServeCommand::invalidAction() expects string, array|bool|string|null given.


(argument.type)